pyproject.toml

```diff
@@ -178,12 +178,10 @@ dependencies = ["tmt[docs]"]
 # files below, but it may be providing additional metadata for RTD to index.
 html = "sphinx-build -b html {root}/docs {root}/docs/_build {args}"
 man = [
-    "cp {root}/docs/header.txt {root}/man.rst",
-    "tail -n+8 docs/overview.rst >> {root}/man.rst",
-    # TODO rst2man cannot process this directive, removed for now
-    "sed '/versionadded::/d' -i {root}/man.rst",
-    "rst2man.py {root}/man.rst > {root}/tmt.1",
-    "rm -f {root}/man.rst",
+    "(cat {root}/docs/header.txt; echo; tail -n+8 {root}/docs/overview.rst) > {root}/docs/man.rst",
+    "sed '/versionadded::/d' -i {root}/docs/man.rst",
```

Comment:
How about adding these to the sphinx setup or the Makefile?
https://github.com/teemtee/tmt/blob/a08a4be5ef73913cab0f7440a400f27667e5f869/docs/conf.py#L316-L327

Probably combining `exclude_patterns = [r"man/*"]` and `manpages` would work. I don't see config options for specific builder overrides.

Reply:
sure, whatever works. This is just a PoC of rendering the manpage in the same way as the current setup, just without rst2man.